src/app/static/src/tests/components/runReport/runReport.test.ts

```diff
@@ -266,6 +266,25 @@ describe("runReport", () => {
         });
     });
 
+    it("report list shows initialReportName prop", async (done) => {
+
+        const wrapper = shallowMount(RunReport, {
+            propsData: {
+                metadata: {
+                    git_supported: true,
+                    instances_supported: true,
+                },
+                initialGitBranches,
+                reportName: "minimal"
+            }
+        });
+
+        setTimeout(() => {
+            expect(wrapper.find("report-list-stub").attributes("initialselectedreport")).toBe("minimal")
```

Comment:
I'd be surprised if you need this timeout, this prop on the child component should be set when mount is completed. Should be able to use `props` for child vue component.

Reply:
Yeah, it's actually needed to show the component in the first place and prop is not visible for this particular component but attributes.